src/blockencodings.cpp

```diff
@@ -43,6 +44,32 @@ uint64_t CBlockHeaderAndShortTxIDs::GetShortID(const uint256& txhash) const {
     return SipHashUint256(shorttxidk0, shorttxidk1, txhash) & 0xffffffffffffL;
 }
 
+namespace {
+    struct ShortIdIndexPair {
+        uint64_t shortid : 48;
+        uint16_t index : 16;
```

Comment:
I think compilers are free to pack them still if they want to, but my recollection is that they aren't required to